laceworksdk/http_session.py

```diff
@@ -32,15 +32,17 @@ class HttpSession:
 
     def __init__(self, account, subaccount, api_key, api_secret, base_domain):
```

Comment:
Why no type hints?

Reply:
So I brought this up when I scoped the Epic. Type hints are great from 3.10 on but in 3.9 they aren't fully baked. Since we are still supporting 3.9 it would be difficult to implement them properly.